apps/console/src/features/provisioning/api/outbound-provisioning.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+import { OutboundProvisioningConfigurationInterface } from "../../applications/models/application";
+import { store } from "../../core/store";
+
+/**
+ * Get an axios instance.
+ */
+const httpClient: HttpClientInstance = AsgardeoSPAClient.getInstance()
+    .httpRequest.bind(AsgardeoSPAClient.getInstance());
+
+export const updateResidentApplicationOutboundProvisioningList = (
+    outboundProvisioningList: OutboundProvisioningConfigurationInterface[]
+): Promise<any> => {
+    const requestConfig: AxiosRequestConfig = {
+        data: {
+            outboundProvisioningIdps: outboundProvisioningList
+        },
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
```

Comment:
Does it make sense to send this header in request, as Access-Control-Allow-Origin is a response header [1]?

[1] https://stackoverflow.com/a/10636765/8810941

Reply:
Addressed 4c98502d24cc3d1a6a564b3468bc1768fd727766